package.json

```diff
@@ -21,7 +22,8 @@
     "prepublish": "npm run build"
   },
   "files": [
-    "dist"
+    "dist",
+    "types"
```

Comment:
Can we do `"*.d.ts",` and place the types in the root of the project?

Reply:
I think we could. I'll do that in the next iteration.

Related the where to place them (@types or here), it should be here. @types if for when a package doesn't include itself